korim/src/commonMain/kotlin/com/soywiz/korim/format/KRA.kt

```diff
@@ -223,15 +223,21 @@ object KRA : ImageFormat("kra") {
                                         layerPartialBitmap,
                                         targetX = layerX,
                                         targetY = layerY,
-                                        name = layerName,
-                                        main = false
+                                        main = false,
+                                        name = layerName
```

Comment:
Was this a cleanuo or something from the IDE to keep params sorted or something?

Reply:
looks like it was auto reformatted from ide